### DISCUSSION TITLE
Options in Choose action can't be re-ordered

### DISCUSSION THREAD

**Discussion by rrooggiieerr** [home-assistant/frontend#17023] [Automations / scripts / scenes] [answered]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When using a Choose action the different options within the Choose action can not be reordered. A workaround is to edit the action in YAML mode and cut-paste the options in the wanted order, however this is not user friendly and error prone.

### Describe the behavior you expected

I expect the different options within the Choose action to be re-ordered using the UI.

Currently a Choose Option can only be deleted. My suggestion would be to replace the trashcan icon with a three dot menu with options to re-order and delete the given option

### Steps to reproduce the issue

1. Create an Automation or Script
2. Add a Choose action
3. Add different options to the Choose action
4. Try to move the options

### What version of Home Assistant Core has the issue?

2023.4.6

**Comment by karwosts** [comment, marked as answer]
Choose options have been reorderable for some time.
